GVFS/GVFS.Service/Mac/NotificationHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿using GVFS.Common;

```

Comment:
`RepoRegistry` class is mostly re-usable on the Mac. But that class has a dependency on `NotificationHandler` - and that is the reason I had to keep its Mac version here.